release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -2764,3 +2764,55 @@ This issue is resolved for:
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1864352[*BZ#1864352*])
+
+* Previously, resource names were getting updated in the edit flow and the edit application user was unable to change the Git repository or update the application. With this fix, the application name is no longer updated white in the edit flow, and the edit flow user is able to change the Git repository and update the application.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1877290[*BZ#1877290*])
+
+* Previously, a watch cache in the Kubernetes API was initialized from the global revision (`etcd`) and could remain for an undefined period if no changes were made. This behavior could lead to a situation in which a client gets a resource version (RV) from a server that has observed a newer RV, disconnects from it due to a network error, and reconnects to a server that is behind, resulting in "Too large resource version" errors. With this release, the reflector is fixed so that it can recover from "Too large resource version" errors and Operators that use the `client-go` library for getting notifications from the server can recover and make progress upon receiving "Too large resource version" errors.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1877346[*BZ#1877346*])
+
+* Previously, when the Authentication Operator received an HTML payload from an OpenID Connect Authentication (OIDC) server that ignored the `Accept: application/json` header, the OIDC servers could respond with an HTML page that states that the authentication Operator failed to parse because it was expecting JSON. Now the Operator ignores the error and does not allow CLI login for OIDC servers that ignore the header.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1879417[*BZ#1879417*])
+
+* Previously, the Image Registry Operator could not get events from the cluster when it encountered "Too large resource version" errors. With this release, the `client-go` library is updated to fix the reflector so that the Operator can recover from "Too large resource version" errors.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1880314[*BZ#1880314*])
+
+* The Kubernetes network proxy does not support multiple cluster CIDRs to detect local traffic. When multiple CIDRs are configured in OpenShift SDN, the Cluster Network Operator (CNO) sets the `KubeProxyConfiguration.clusterCIDR` field to an empty string. In {product-title} version 4.4 and earlier, the empty value was ignored, but in version 4.5 and later, passing an empty value causes an error. As a result, after upgrading from 4.4 to 4.5, if the `sdn-config ConfigMap` has an empty string in the `clusterCIDR` field, the configuration cannot be parsed and the SDN pods enter a crash loop. With this release, the empty value is ignored and SDN pods no longer crash when multiple CIDRs are configured.
```

Comment:
s/{product-title} version 4.4/{product-title} 4.4/ 
And you can remove "version" later in the same sentence.